ansible/aix61-standalone/manualBootstrap.md

```diff
@@ -12,16 +12,114 @@ chfs -a size=+250000 /
 chfs -a size=917504 /tmp
 ```
 
-AIX72: unneeded: -1 and -2 and aix7-2 have different sizes, and 7.2 is big
-enough already:
+For the systems without ramdisks `/home` needs to be set to the same size as `/home/iojs/build` would be on a ramdisk machine
+
+```bash
+chfs -a size=23000000 /home
+```
+
+## Edit SSH config
+
+Add the following two lines to `/etc/ssh/sshd_config`:
+
+```
+PasswordAuthentication no
+ChallengeResponseAuthentication no
+```
+
+## Remove failedlogin file
+
+If /etc/security/failedlogin is growing without bounds on AIX6.1, remove
+it with a cron job, use `crontab -e`, and add one line:
+```sh
+0 12 * * * /usr/bin/rm -f /etc/security/failedlogin
+```
+
+## Fix "Missing" shared objects
+
+On the 7.1 machines we were facing this issues when yum installed packages were not able to find some of the shared objects they needed
+
+```sh
+bash-5.0$ gmake -v
+exec(): 0509-036 Cannot load program gmake because of the following errors:
+        0509-022 Cannot load module /opt/freeware/lib/libintl.a(libintl.so.8).
+        0509-150   Dependent module /usr/lib/libiconv.a(libiconv.so.2) could not be loaded.
+        0509-152   Member libiconv.so.2 is not found in archive
+        0509-022 Cannot load module make_64.
+        0509-150   Dependent module /opt/freeware/lib/libintl.a(libintl.so.8) could not be loaded.
+        0509-022 Cannot load module .
+```
+
+The fix is as following:
+
+```sh
+sudo rm /usr/lib/libiconv.a && sudo ln -s /opt/freeware/bin/libiconv.a /usr/lib
+```
+
+# AIX 7.2 Install
+
+Most packages should be installed via ansible.
+
+If there are any missing they should be installed via yum
+
+What you do need to install manually is **ccache**
+
+
+```bash
+mkdir -p /opt/gcc-6.3 && cd /opt/gcc-6.3
+curl -L https://ci.nodejs.org/downloads/aix/gcc-6.3-aix7.2.ppc.tar.gz | /opt/freeware/bin/tar -xzf -
+```
+
+## ccache 3.7.4 on AIX 7.2
+
+```bash
+mkdir -p /opt/ccache-3.7.4 && cd /opt/ccache-3.7.4
+curl -L https://ci.nodejs.org/downloads/aix/ccache-3.7.4.aix7.2.ppc.tar.gz | /opt/freeware/bin/tar -xzf -
+```
```

Comment:
Missing blank line before the next section.

Reply:
good catch